src/Package/Impl/DataInspect/DataImport/EnterUrl.xaml.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.IO;
+using System.Net;
+using System.Threading.Tasks;
+using System.Windows;
+using Microsoft.Common.Core;
+using Microsoft.VisualStudio.PlatformUI;
+
+namespace Microsoft.VisualStudio.R.Package.DataInspect.DataImport {
+    /// <summary>
+    /// Interaction logic for ImportDataWindow.xaml
+    /// </summary>
+    public partial class EnterUrl : DialogWindow {
+        public EnterUrl() {
+            InitializeComponent();
+        }
+
+        public string DownloadFilePath { get; private set; }
+
+        public string Name { get; private set; }
+
+        public void DeleteTemporaryFile() {
+            if (!string.IsNullOrEmpty(DownloadFilePath)) {
+                try {
+                    if (File.Exists(DownloadFilePath)) {
```

Comment:
No reason to call `File.Exists` - access to hard drive is much slower than stack trace collection